deploy/olm-catalog/csv-config.yaml

```diff
@@ -1,7 +1,7 @@
-operator-path: deploy/operator.yaml
+operator-path: REPLACE_ROOT_PATH/deploy/operator.yaml
```

Comment:
where does this `REPLACE_ROOT_PATH` come from?

Reply:
https://github.com/codeready-toolchain/api/pull/109/files#diff-8f3e5d8ae2c233d5432bf1c852317cf3R181-R182